### PR TITLE
[ZT] Add agentless options

### PR DESCRIPTION
**Pull request by ranbel**

### CODE REVIEW

**Review by kennyj42 — APPROVED** [review]

File: content/cloudflare-one/connections/connect-devices/agentless/_index.md [hunk 1]
```diff
@@ -1,13 +1,16 @@
...
 
 # Agentless options
 
-If you are unable to install the WARP client on your devices (for example, Windows Server does not support the WARP client), you can use agentless options to perform a subset of Gateway functions.
-
-- **[DNS filtering](/cloudflare-one/connections/connect-devices/agentless/dns/)** without WARP requires changing your local DNS resolver to Cloudflare.
-- **[HTTP filtering](/cloudflare-one/connections/connect-devices/agentless/pac-files/)** without WARP requires configuring your browser to forward traffic to a Cloudflare proxy server.
+If you are unable to install the WARP client on your devices (for example, Windows Server does not support the WARP client), you can use agentless options to enable a subset of Zero Trust features.
 
+- **[Gateway DNS policies](/cloudflare-one/connections/connect-devices/agentless/dns/)**
```
Comment (kkrum) [inline content/cloudflare-one/connections/connect-devices/agentless/_index.md:11]: Without Identity, just like HTTP policies

Comment (ranbel) [reply content/cloudflare-one/connections/connect-devices/agentless/_index.md:11]: Does this count as identity? https://developers.cloudflare.com/cloudflare-one/connections/connect-devices/agentless/dns/dns-over-https/#filter-doh-requests-by-user

Comment (kkrum) [reply content/cloudflare-one/connections/connect-devices/agentless/_index.md:11]: Hmm good catch. And  https://developers.cloudflare.com/cloudflare-one/connections/connect-devices/agentless/dns/locations/ for that matter which is also an identity.

File: content/cloudflare-one/connections/connect-devices/agentless/_index.md [hunk 2]
```diff
@@ -1,13 +1,16 @@
...
+- **[Gateway HTTP policies](/cloudflare-one/connections/connect-devices/agentless/pac-files/)** without user identity, device posture, Data Loss Prevention, or antivirus scanning
```
Comment (kkrum) [inline content/cloudflare-one/connections/connect-devices/agentless/_index.md:12]: DLP and AV are supported via agentless as they don't rely on anything from WARP

Comment (ranbel) [reply content/cloudflare-one/connections/connect-devices/agentless/_index.md:12]: updated in https://github.com/cloudflare/cloudflare-docs/pull/9799

**Pull request merged**